### ISSUE TITLE
wrongful usage of create course API

### ISSUE THREAD

**Issue by vkarakolis-epu** [QualiChain/fe#381]
Please check again this APIs when creating and editing courses. We get an error when trying to edit a course and we think it because of a wrongly named parameter. The correct field is called academic_organisation_id. Please check your calls to the following APIs.

**Comment by mmilaprat** [comment]
FE updated, but this field has been changed, no? I think it was academic_organisation in the past.

**Comment by vkarakolis-epu** [comment]
> FE updated, but this field has been changed, no? I think it was academic_organisation in the past.

Yes you are right

**Comment by sskalidakis** [comment]
Yes, @mmilaprat . Please, change the name of this field in any call you make for creating or updating a course from "academic_organisation" to "academic_organisation_id".

**Comment by mmilaprat** [comment]
done in MR 385

**Comment by sskalidakis** [comment]
> done in MR 385

Yeap! Sorry about that. Needed to clean my cache! Works fine!
